src/Viewer/services/ActionHandler.js

```diff
@@ -121,6 +121,25 @@ class ActionHandler {
         this._updateStateCallback(CLP_WORKER_PROTOCOL.UPDATE_STATE, this._logFile.state);
     };
 
+    /**
+     * Goes to the specified log event with the specified timestamp.
+     * Go to new page if needed.
+     * @param {number} timestamp
+     */
+    changeEventWithTimestamp (timestamp) {
+        if (!isNumeric(timestamp)) {
+            throw (new Error("Invalid timestamp provided."));
+        }
+        this._logFile.state.logEventIdx = this._logFile.getLogEventIdxFromTimestamp(timestamp);
```

Comment:
I think the newest private class variable syntax is using #. I'm not sure if the _ prefix here means something should be private.
https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Classes/Private_class_fields